prov/efa/src/rxr/rxr_pkt_type_req.c

```diff
@@ -853,6 +886,18 @@ ssize_t rxr_pkt_proc_matched_rtm(struct rxr_ep *ep,
 		rx_entry->cq_entry.len = rx_entry->total_len;
 
 	pkt_type = rxr_get_base_hdr(pkt_entry->pkt)->type;
+
+	if (pkt_type >= RXR_DC_REQ_PKT_BEGIN &&
+	    pkt_type <= RXR_BASELINE_DC_REQ_PKT_END)
```

Comment:
Should not it be "<" (not "<=")?

Reply:
I think we use `<=` for boundary check against `pkt_type` (https://github.com/ofiwg/libfabric/blob/master/prov/efa/src/rxr/rxr_pkt_cmd.c#L702)

Reply:
You defined DC_PKT_END as one beyond the maximum possible value, right? The maximum possible packet ID for DC is 141 (RXR_DC_WRITE_RTA_PKT). You defined RXR_DC_PKT_END as 142. So you should use "<" here.

Reply:
Ok I will update it.

Reply:
Removed the equal signs, resolving.